packages/serialize/types/index.d.ts

```diff
@@ -1,19 +1,26 @@
 // Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
 // TypeScript Version: 2.8
-
-import { RegisteredCache, SerializedStyles } from '@emotion/utils'
+import type { RegisteredCache } from '@emotion/cache'
 import * as CSS from 'csstype'
 
-export { RegisteredCache, SerializedStyles }
+export interface SerializedStyles {
+  name: string
+  styles: string
+  map?: string
+  next?: SerializedStyles
+}
 
 export type CSSProperties = CSS.PropertiesFallback<number | string>
 export type CSSPropertiesWithMultiValues = {
   [K in keyof CSSProperties]:
+    | null
+    | undefined
+    | false
     | CSSProperties[K]
     | Array<Extract<CSSProperties[K], string>>
 }
 
-export type CSSPseudos = { [K in CSS.Pseudos]?: CSSObject }
+export type CSSPseudos = { [K in CSS.Pseudos]?: CSSInterpolation }
```

Comment:
- [a test](https://github.com/emotion-js/emotion/blob/f3b268f7c52103979402da919c9c0dd3f9e0e189/packages/react/__tests__/css.js#L39) that allows CSS composition but not in TS
- [a test](https://github.com/emotion-js/emotion/blob/f3b268f7c52103979402da919c9c0dd3f9e0e189/packages/react/__tests__/css.js#L67) that allows `false` value but not in TS
- [a test](https://github.com/emotion-js/emotion/blob/f3b268f7c52103979402da919c9c0dd3f9e0e189/packages/react/__tests__/css.js#L250) that allows child selector array but not in TS

Reply:
Looks legit but I will have to reexamine this more closely later.